src/extension.ts

```diff
@@ -81,32 +83,40 @@ export async function activate(context: vscode.ExtensionContext) {
   let { repoName, ownerUsername } = await getRepoInfo();
   repo = repoName;
   owner = ownerUsername;
+  reporter.sendTelemetryEvent("repoInfo", { owner, repo });
```

Comment:
Would it be better to add {owner, repo} to each one of the events, instead of tracking it as a separate event?

Reply:
This would allow tracking the user all the way. 
I'd leave this one as it's own starting point (as in "extensionActivated" we don't have the info).